### ISSUE TITLE
[Code Quality] Execute Prettier for Code Formatting

### ISSUE THREAD

**Issue by Saumya40-codes** [Programming-Club-Ahmedabad-University/WOC-2024#4]
Ensure consistent code formatting across the repository by setting up and executing Prettier.

### Approach:
- Install Prettier as a dev dependency. Configure Prettier with a .prettierrc file.
     ```
     npm install --save-dev prettier
     ```

- Create a file named `.prettierrc ` in root path
  - A very minimalistic configuration we can go with is the following (i.e. the content of .prettierrc  file)
    ```
    {
	"tabWidth": 2
    }
    ```

- Format all existing codebase files.
  - Run:
    ```
    npx prettier --write .
    ```

This will make all the files we have follow a standard and appropriate indentation 

Learn more about [prettier](https://prettier.io/docs/en/) from here

**Comment by KushalXCoder** [comment]
Can I work on this ?

**Comment by KushalXCoder** [comment]
Also, one more thing, should I also add .prettierignore file which would contain node_modules, .env, and other files etc.

**Comment by Saumya40-codes** [comment]
> Also, one more thing, should I also add .prettierignore file which would contain node_modules, .env, and other files etc.

Yes, good catch. We wont be needing prettier for node_modules and .env files.
